sources/packages/api/src-sql/SINValidations/Create-sin-validations.sql

```diff
@@ -0,0 +1,64 @@
+CREATE TABLE IF NOT EXISTS sims.sin_validations (
+  id SERIAL PRIMARY KEY,
+  date_sent TIMESTAMP WITH TIME ZONE,
+  date_received TIMESTAMP WITH TIME ZONE,
+  file_sent VARCHAR(50),
+  file_received VARCHAR(50),
+  given_name_sent VARCHAR(50),
+  surname_sent VARCHAR(50),
+  dob_sent Date,
+  request_status_code VARCHAR(2),
```

Comment:
Can we have a boolean column to indicate the "interpretation" of the current validation as true/false?

Reply:
i thought of more like this table captures the data as we get it in the file, I can change it. Do we really need to??

Reply:
Capturing the data as it is on the file is good for troubleshooting. For the system, we just need something to tell us Valid/NotValid.

Whatever we need to check if the SIN is valid we need to check in the below way, what is also valid but the developer need to know the matchStatusCodes is what we should look into and that the code is "01".
```
sinValidation.matchStatusCodes === MatchStatusCodes.successfulMatch
```
We can leave as it is.

Reply:
Yeah i get it, but as we are already doing it to check for valid SIN, I just wanted to capture what is received.
<img width="795" alt="image" src="https://user-images.githubusercontent.com/62901416/156839522-46a47df4-625b-4c67-b2e5-c1c0dc26e7e8.png">